site/_docs/reference.md

```diff
@@ -2783,7 +2783,9 @@ BigQuery's type system uses confusingly different names for types and functions:
 | h s | PARSE_URL(urlString, partToExtract [, keyToExtract] ) | Returns the specified *partToExtract* from the *urlString*. Valid values for *partToExtract* include HOST, PATH, QUERY, REF, PROTOCOL, AUTHORITY, FILE, and USERINFO. *keyToExtract* specifies which query to extract
 | b | POW(numeric1, numeric2)                        | Returns *numeric1* raised to the power *numeric2*
 | b | REGEXP_CONTAINS(string, regexp)                | Returns whether *string* is a partial match for the *regexp*
+| b | REGEXP_EXTRACT(string, regexp[, position[, occurrence]]) | Returns the substring in *string* that matches the *regexp*. Returns `NULL` if there is no match. Use *position* for the start index of search range and *occurrence* for the specific occurence of match in *string*
```

Comment:
Don't think NULL should be quoted based on other usages. 

Do you think you could rewrite to sort of match the `INSTR` description? That might be more simple and is a good example of a function with optional arguments. Let me know what you think.

Reply:
Thanks Tanner, but I also saw a couple of instances (MAP_CONCAT, ARRAY_CONCAT) where it was `NULL`, and another few places with returns null in lower case or NULL in upper, what should be the standard here a bit confused.

Reply:
oh interesting. in that case i guess either is fine but it should probably be standardized.